actions/post_actions.jsx

```diff
@@ -317,6 +317,32 @@ export function hideEditPostModal() {
     };
 }
 
+export function deleteCombinedPost(combinedPost) {
```

Comment:
I forgot where we ended up with moving this to the redux repo. Are you going to be doing that, or did we decide against it?

Reply:
Yeah, I'm going to do that - move to redux. I just addressed first the issues from PM. After that,  I'll deal with code comments. Some of you and EN's comments were implemented but not all (including this).

Reply:
Logics were moved to redux.